src/utils/LayerDropdown.tsx

```diff
@@ -0,0 +1,92 @@
+import {
```

Comment:
This file is a component, it should be under the `src/components` tree somewhere...